src/User32.Desktop/User32.cs

```diff
@@ -68,6 +69,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowLong(IntPtr hWnd, WindowLongIndexFlags nIndex);
 
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
```

Comment:
This is another case in with the caller is supposed to send a character buffer for the function to modify and respond with the Class Name. I am not sure there are much other alternatives to StringBuilder on these cases. As far as I had read and understood, all functions that expect a character buffer as parameter and changes it inside should be marshalled as StringBuilder.